File: Python/tests/test_prl_rp.py
import pytest

from hbayesdm.models import prl_rp


def test_prl_rp():
    _ = prl_rp(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_prl_rp.R
context("Test prl_rp")
library(hBayesDM)

test_that("Test prl_rp", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(prl_rp(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
